src/ansys/simai/core/utils/misc.py

```diff
@@ -34,3 +34,11 @@ def build_boundary_conditions(boundary_conditions: Optional[Dict[str, Any]] = No
     if not bc:
         raise ValueError("No boundary condition was specified.")
     return bc
+
+
+def dict_get(obj: dict, *keys: str, default=None):
+    """Get the requested key of the dictionary or opt to the default."""
+    for k in keys:
+        if k:
+            obj = obj.get(k, {}) or {}
```

Comment:
why `if k` ?
```suggestion
        obj = obj.get(k, {}) or {}
```

Reply:
oh, my bad; the assignment in `ModelConfiguration._to_payload` had a bug on the assignment. If clause is removed.